frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/StatusIconAndText.tsx

```diff
@@ -18,7 +17,7 @@ const StatusIconAndText: React.FC<StatusIconAndTextProps> = ({
   className,
 }) => {
   if (!title) {
-    return <>{DASH}</>;
+    return <>{'-'}</>;
```

Comment:
why not continue with const?

Reply:
@invincibleJai  We definitely can, just not sure why we need also a const for something like a dash. I thought it is more some kind of a remnant from the past.

Reply:
@andrewballantyne gotha, sounds good, thank you :slightly_smiling_face: